setup.py

```diff
@@ -1,6 +1,25 @@
 #!/usr/bin/env python
 """The setup script."""
 from setuptools import find_packages, setup
+from setuptools.command.install import install
+
+
+class InstallWithCompile(install):
```

Comment:
This will need to be translated somehow when we adopt `pyproject.toml` conventions. It shouldn't be difficult, but given that we're reliant on a bona fide `setuptools` function, we might need to keep `setuptools` as our build backend.

We'll cross that bridge when we get there.